docs/sources/getting-started/getting-started-prometheus.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Getting started with Grafana and Prometheus"
+description = "Guide for getting started with Grafana"
+keywords = ["grafana", "intro", "guide", "started"]
+type = "docs"
+aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
+[menu.docs]
+identifier = "getting_started-grafana-prometheus"
+parent = "guides"
+weight = 300
++++
+
+# Getting Started with Grafana and Prometheus
+
+Prometheus is an open-source systems monitoring and alerting toolkit for which Grafana includes built-in support. This topic walks you through the steps to create a series of dashboards in Grafana displaying system metrics of a host monitored by Prometheus.
+
+You can also configure your Grafana instance in the cloud to display system metrics. For more information, refer to the [Grafana Cloud](https://grafana.com/docs/grafana-cloud/) documentation.
+
+## Step 1: Install Grafana and build your first dashboard
+
+Use the instructions in [Getting started with Grafana]({{< relref "getting-started.md" >}}) to:
+- Install Grafana
+- Login to Grafana
```

Comment:
```suggestion
- Log in to Grafana
```

Reply:
Each of these bullets is a sentence. Please add periods.

Reply:
Please address this change.

Reply:
Done